### DISCUSSION TITLE
SideDownload Error

### DISCUSSION THREAD

**Discussion by esshor** [aclap-dev/video-downloadhelper#1115] [Help and Q&A]
### Your question

Since the recent update of the extension and its logo icon, I haven't been able to successfully download any streaming video at all. All start, then very quickly end with an error.
This was what the most recent attempt error displayed:

"SideDownload error:
[https @ 000002652bf852c0] Stream ends prematurely at 236865, should be 508728
[https @ 000002652bf85f00] Stream ends prematurely at 142656, should be 1317880
[https @ 000002652bf85b00] Stream ends prematurely at 146751, should be 526776
[https @ 000002652bf852c0] Stream ends prematurely at 11584, should be 559864
[aac_adtstoasc @ 000002652bf7bd80] Error parsing ADTS frame header!
[mp4 @ 000002652cd9ffc0] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input
[aost#0:1/copy @ 000002652c77d840] Error submitting a packet to the muxer: Invalid data found when processing input
[out#0/mp4 @ 000002652c5ab9c0] Error muxing a packet"

Is anyone able to identify from this what the problem is? And what I can do to remedy it?

### Addon & CoApp version and details

version: 9.0.2.12
channel: stable
build date: 2024-05-30
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): c164a48ce2994d91
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Admin",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Let's have at least one URL of a web page that hosts one of your problematic videos.  It must be on a web site where I do not need to register a user ID & password.
